model/accounts_mgmt/v1/subscription_notify_type.model

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+struct SubscriptionNotify {
+	BccAddress string
+	Subject string
+	TemplateName string
+	TemplateParameters []TemplateParameter
```

Comment:
Please add comments describing what each attribute is. This is used by the build scripts to add descriptions to the OpenAPI spec.